pygenometracks/tracksClass.py

```diff
@@ -356,10 +356,16 @@ def parse_tracks(self, tracks_file):
                     except ValueError:
                         log.error("In section {}, {} was set to {}"
                                   " whereas we should have a boolean value."
-                                  "Please, use "
-                                  "{}".format(section_name, name, value,
-                                              str([k for k in parser.BOOLEAN_STATES])))
-                        exit()
+                                  "Please, use true or false."
+                                  "".format(section_name, name, value))
+                        exit(1)
+                    if value not in ['true', 'false']:
```

Comment:
@LeilyR , you would change it to: `if value.lower() not in ['true', 'false']:`?

Reply:
yes, sounds like a good idea to me.

Reply:
Done